### PR TITLE
First Commit

### PR DESCRIPTION
**Pull request by cs-shubham-morankar**
#### Descriptions: Following operations has been implemented:

- Create Spreadsheet
- Get Spreadsheet Details
- Filter Spreadsheet
- Add Row to a Spreadsheet
- Get Spreadsheet Rows
- Update Rows in a Spreadsheet
- Update Rows in a Spreadsheet by Filter
- Clear Rows from a Spreadsheet
- Clear Rows from Spreadsheet by Filter
- Move Sheet

#### UTCs:

- [x] Verified check health.
- [x] Verified all operations.
- [x] Verified all playbooks with it's tags, inactive state, and debug mode off.

### CODE REVIEW

File: google-sheets/info.json [hunk 1]
```diff
@@ -0,0 +1,1148 @@
...
+          "type": "text",
+          "visible": true,
+          "editable": true,
+          "required": false,
+          "tooltip": "Specify the title of the spreadsheet based on which you want to create."
+        },
+        {
+          "title": "Locale/Language",
+          "name": "locale",
+          "description": "Specify the locale of the spreadsheet in one of the following formats: en, fil, or en_US. Note: When updating this field, not all locales/languages are supported.",
```
Comment (cs-omkar-kumbhar) [inline google-sheets/info.json:102]: if possible you can specify the link where user can find all locales/languages

File: google-sheets/info.json [hunk 2]
```diff
@@ -0,0 +1,1148 @@
...
+          "name": "autoRecalc",
+          "description": "Select the amount of time to wait before volatile functions are recalculated. You can choose from the following options: RECALCULATION_INTERVAL_UNSPECIFIED, ON_CHANGE (default), MINUTE, or HOUR.",
+          "type": "select",
+          "visible": true,
+          "editable": true,
+          "required": false,
+          "tooltip": "Select the amount of time to wait before volatile functions are recalculated. You can choose from the following options: RECALCULATION_INTERVAL_UNSPECIFIED, ON_CHANGE (default), MINUTE, or HOUR",
+          "options": [
+            "RECALCULATION_INTERVAL_UNSPECIFIED",
+            "ON_CHANGE",
```
Comment (cs-omkar-kumbhar) [inline google-sheets/info.json]: specify the options in this format 'On Change'

File: google-sheets/info.json [hunk 3]
```diff
@@ -0,0 +1,1148 @@
...
+          "title": "Value Input Option",
+          "name": "valueInputOption",
+          "description": "Select the input data should be interpreted. You can choose from the following options: INPUT_VALUE_OPTION_UNSPECIFIED, RAW, or USER_ENTERED.",
+          "type": "select",
+          "visible": true,
+          "editable": true,
+          "required": true,
+          "tooltip": "Select the input data should be interpreted. You can choose from the following options: INPUT_VALUE_OPTION_UNSPECIFIED, RAW, or USER_ENTERED.",
+          "options": [
+            "INPUT_VALUE_OPTION_UNSPECIFIED",
```
Comment (cs-omkar-kumbhar) [inline google-sheets/info.json]: specify the options in this format 'On Change'

File: google-sheets/info.json [hunk 4]
```diff
@@ -0,0 +1,1148 @@
...
+          "title": "Insert Data Option",
+          "name": "insertDataOption",
+          "description": "Select the input data should be inserted. You can choose from the following options: OVERWRITE or INSERT_ROWS.",
+          "type": "select",
+          "visible": true,
+          "editable": true,
+          "required": false,
+          "tooltip": "Select the input data should be inserted. You can choose from the following options: OVERWRITE or INSERT_ROWS.",
+          "options": [
+            "OVERWRITE",
```
Comment (cs-omkar-kumbhar) [inline google-sheets/info.json]: specify the options in this format 'On Change'

File: google-sheets/info.json [hunk 5]
```diff
@@ -0,0 +1,1148 @@
...
+          "title": "Response Value Render Option",
+          "name": "responseValueRenderOption",
+          "description": "Determines how values in the response should be rendered. You can choose from the following options: FORMATTED_VALUE, UNFORMATTED_VALUE, or FORMULA. The default render option is FORMATTED_VALUE.",
+          "type": "select",
+          "visible": true,
+          "editable": true,
+          "required": false,
+          "tooltip": "Determines how values in the response should be rendered. You can choose from the following options: FORMATTED_VALUE, UNFORMATTED_VALUE, or FORMULA. The default render option is FORMATTED_VALUE.",
+          "options": [
+            "FORMATTED_VALUE",
```
Comment (cs-omkar-kumbhar) [inline google-sheets/info.json]: specify the options in this format 'On Change'

File: google-sheets/info.json (hunk 1 above)
Comment (cs-shubham-morankar) [reply google-sheets/info.json:102]: We don't have any table or link.

**Comment by cs-shubham-morankar** [comment]
Development under process as per new requirement.

**Pull request closed without merging**